ReactNative/ReactTwitter/core/views/debug/debug-uitesting.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+import React from 'react'
+import {
+  StyleSheet,
+  View,
+  Navigator,
+  Text
+} from 'react-native'
+
+var Button = require('react-native-button')
+
+var DebugUITestingScene = React.createClass({
+  statics: {
+    navigatorID () {
+      return 'debug-uitesting-ios-scene-identifier'
+    },
+    navigatorTitle () {
+      return 'iOS UI Testing'
```

Comment:
Well, it's iOS only because we coudn't find a way to have the  automated UI tests working on Android, but the page would actually be the same if they were working isn't it?
My point is that the tests are not dependent on that page, but in the native part, so that page should have a common name.
This is just my opinion, and this is not a blocking comment, so if you prefer to keep it, no problem